### DISCUSSION TITLE
DAMP in HDR mode?

### DISCUSSION THREAD

**Discussion by czukowski** [gopro/labs#660] [Q&A]
Does DAMP have an effect when recording video in HDR mode? I'm inclined to believe that it probably doesn't but I would like a confirmation on it. Thanks!

From the [description](https://gopro.github.io/labs/control/extensions/):

> DAMP=0.1 to 1000 - Control over the auto-exposure damping. When moving a camera through dramatically changing lighting, like biking through a forest, in and out of sunlight, or flying a drone through a short tunnel, the exposure will adjust automatically, correctly in most scenarios, but sometimes not optimal for some rapidly changing light levels. The camera’s auto-exposure currently takes about one second to adjust from sunlight to indoor conditions, but if you are flying a drone through indoors for only a few seconds, the camera would over-expose on exit. In this scenario using an exposure lock might be preferable, maintaining an outdoor exposure throwout, but exposure lock is risky for shooting under variable lighting conditions. Changing the damping might be what you need. Setting the DAMP=1 is the default, so setting to 10 would slow the camera’s exposure adjustments 10X. Now shooting into dark for moment will not cause much over-exposure (if any). Setting DAMP=60 will take a minute for exposure changes. Setting DAMP lower than 1, like 0.2, would make it adjust faster. There is no perfect setting, this is just more control.

**Comment by dnewman-gpsw** [comment]
Have you tried it?

**Reply by czukowski** [reply]
I didn't do special testing of this setting, I've just tried to set 5 instead of 1 and do a shooting but it was at the same time as I switched to using HDR. I can see how the levels are adjusted when moving in or out of the shadows, but it seems fairly fast anyway, that's why I'm asking, if it's supposed to make a difference or not. Also exposure controls on the camera are disabled in HDR mode.

If you don't know, then I'll make some time to do the testing.

**Reply by dnewman-gpsw** [reply]
Hacks might not work in same new modes, but we won't know unless someone tries it.  I haven't tested this combo.

**Reply by czukowski** [reply]
It wasn't very sunny in the morning, but I've made a few attempts. There are 2 series with 4 clips in each, shot in order with DAMP=0.1, DAMP=1, DAMP=5 and DAMP=10.

1. https://gopro.com/v/Bb9PgK1pwKrOQ
2. https://gopro.com/v/vkDn3Vka8Evvn

It would seem to me that the adjustment speed is the same overall, however with higher values, it has apparent "jumps", especially visible in Gx010412.mp4 and Gx010413.mp4...
